src/briefcase/console.py

```diff
@@ -71,32 +71,47 @@ class RichConsoleHighlighter(RegexHighlighter):
 class Printer:
     """Interface for printing and managing output to the console and/or log."""
 
-    # Console to manage console output.
-    console = RichConsole(
-        highlighter=RichConsoleHighlighter(), emoji=False, soft_wrap=True
-    )
+    def __init__(self, width=180):
+        """Create an interface for printing and managing output to the console and/or
+        log.
 
-    # Console to record all logging to a buffer while not printing anything to the console.
-    # We need to be wide enough to render `sdkmanager --list_installed` output without
-    # line wrapping.
-    LOG_FILE_WIDTH = 180
-    # Rich only records what's being logged if it is actually written somewhere;
-    # writing to /dev/null allows Rich to do so without needing to print the logs
-    # in the console or save them to file before it is known a file is wanted.
-    dev_null = open(os.devnull, "w", encoding="utf-8", errors="ignore")
-    log = RichConsole(
-        file=dev_null,
-        record=True,
-        width=LOG_FILE_WIDTH,
-        no_color=True,
-        markup=False,
-        emoji=False,
-        highlight=False,
-        soft_wrap=True,
-    )
+        The default width is wide enough to render the output of ``sdkmanager
+        --list_installed`` without line wrapping.
 
-    @classmethod
-    def __call__(cls, *messages, stack_offset=5, show=True, **kwargs):
+        :param width: The width at which content should be wrapped.
```

Comment:
nitpick: the uninitiated may be misled by this naming and description of `width`. It sounds like it'll affect both what's printed to the console and the log....while it's only relevant to the log.

Reply:
Agreed- I've tweaked the naming.